ember-app/tests/unit/services/board-syncing-test.js

```diff
@@ -15,7 +15,7 @@ moduleFor('service:board-syncing', {
 
 test('syncs the boards issues successfuly', (assert)=>{
   var issues = ['issue1', 'issue2'];
-  var success = $.ajax().then(()=>{return issues});
+  var success = $.ajax().then(()=>{return issues;});
```

Comment:
Yep, For inline functions I still lean towards explicit `return` declarations... for readability sake more than anything (this isn't ruby after all :D )